framework/generated/khronos_generators/vulkan_generators/vulkan_struct_to_json_header_generator.py

```diff
@@ -82,25 +86,17 @@ def beginFile(self, genOpts):
             #include "decode/custom_vulkan_struct_to_json.h"
 
             GFXRECON_BEGIN_NAMESPACE(gfxrecon)
-            GFXRECON_BEGIN_NAMESPACE(decode)
-            ''')
-        body += "\n"
+            GFXRECON_BEGIN_NAMESPACE(decode)''')
```

Comment:
Does this not change whitespace in the output?  Seems like it should, but there's no h/cpp diff in the commit.  Hmmm...